package.json

```diff
@@ -149,10 +154,14 @@
 		"vite-plugin-wasm": "^3.2.2",
 		"vite-tsconfig-paths": "^4.0.2",
 		"vitepress": "1.0.0-alpha.60",
-		"vitest": "^0.25.8"
+		"vitest": "^0.25.8",
+		"vscode": "1.1.37",
+		"yargs": "^17.7.1"
 	},
 	"workspaces": [
 		"packages/nx-extensions",
+		"packages/wp-now",
```

Comment:
cc @sejas

Reply:
- I was assuming this was missing. https://github.com/WordPress/wordpress-playground/pull/284/files
Thanks for adding it 🙏 .